### ISSUE TITLE
Newgrounds Sim v 1.2 - Panic when filling out file details

### ISSUE THREAD

**Issue by anpShawn** [ruffle-rs/ruffle#1489]
https://www.newgrounds.com/portal/view/236362
First, when leaving the game sit on the file menu, the background music will stop after one loop

Next, when starting a new file, the game prompts you to fill in some "profile information"
This screen makes use of a prebuilt flash "dropdown" component where you can select your gender. This dropdown is blank, however, and clicking it produces no behavior. If you attempt to continue to the next screen the game will display this error:

```
# Error Info
Error name: Error
Error message: panicked at 'already mutably borrowed: BorrowError', /Users/michaelwelsh/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:54:21
Error stack:
```
Error: panicked at 'already mutably borrowed: BorrowError', /Users/michaelwelsh/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:54:21
    at Object.Y (https://www.newgrounds.com/js/ruffle/lib/core.ruffle.59944369d57cefbd96e3.js:1:20276)
    at __wbg_new_df6e6ab7a65c4c4d (<anonymous>:1:29625)
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[1136]:0x214239
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[1355]:0x23c4dd
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[3574]:0x2e7ac8
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[2819]:0x2cb249
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[3869]:0x2ec71a
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[3899]:0x2ecc82
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[3777]:0x2eb3f3
    at https://www.newgrounds.com/js/ruffle/lib/f55597c39d793a933f5c.module.wasm:wasm-function[3900]:0x2eccb8
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://www.newgrounds.com/projects/games/50282/preview/filetype/0

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36
OS: Win32

# Ruffle Info
Ruffle version: 0.1.0
Ruffle source: undefined
```

**Comment by Toad06** [comment]
Thanks for this bug report!

The main issue is now fixed, it was related to text fields (PR #1602).
The dropdown menu also seems to work correctly now.

The `Sound.onSoundComplete` listener is used to loop the background music, but it's currently unimplemented (cc #273 and #396).

**Comment by Herschel** [comment]
Background music loop fixed by #2825. Looks like this is working now.
